### PR TITLE
Tests and refactor for the send-tasks script, script_util helper functions

### PR DESCRIPTION
**Pull request by y0urself**
**What**:

Refactoring `gvm-tools` scripts:
* adding a `script_utils.py` file for reusable script functions
* refactored `send_tasks.py` and added unittest

**Why**:

* less, reusable code is easier to understand ...
* We have no tests for our scripts

**How**:

**Checklist**:

- [x] Tests
- [x] [CHANGELOG](https://github.com/greenbone/gvm-tools/blob/master/CHANGELOG.md) Entry
- [ ] Documentation

### CODE REVIEW

**Comment by y0urself** [comment]
@dependabot rebase

**Review by bjoernricks — CHANGES_REQUESTED** [review]
`send-task.gmp.py` still got renamed to `send-task.py`. I think there is already a module for common script code `helper.py`and your code from `script_utils.py` should go into that module. Personally I am not a fan of using underscores in Python module names. For me it looks at bit weird in imports.

**Comment by y0urself** [comment]
> `send-task.gmp.py` still got renamed to `send-task.py`. I think there is already a module for common script code `helper.py`and your code from `script_utils.py` should go into that module. Personally I am not a fan of using underscores in Python module names. For me it looks at bit weird in imports.

I looked into `helper.py`. There were only functions used by the gvm-tools module, not by scripts, but prove me wrong. 
Every script used their own functions with massive code duplications. This is why I wanted to create utilities that are reusable.

**Comment by bjoernricks** [comment]
> > `send-task.gmp.py` still got renamed to `send-task.py`. I think there is already a module for common script code `helper.py`and your code from `script_utils.py` should go into that module. Personally I am not a fan of using underscores in Python module names. For me it looks at bit weird in imports.
> 
> I looked into `helper.py`. There were only functions used by the gvm-tools module, not by scripts, but prove me wrong.
> Every script used their own functions with massive code duplications. This is why I wanted to create utilities that are reusable.

If I remember correctly I did add the Table class for some script to remove an additional dependency.

File: tests/scripts/test_send_tasks.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+
+import unittest
+import os
+from unittest.mock import patch, MagicMock
+from pathlib import Path
+from lxml import etree
+from . import GmpMockFactory, load_script
+
+
+CWD = os.path.abspath(os.path.join(__file__, '../'))
```
Comment (bjoernricks) [inline tests/scripts/test_send_tasks.py:28]: Would be better to use Path from pathlib here

File: tests/scripts/test_send_tasks.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+
+class SendTasksTestCase(unittest.TestCase):
+    def setUp(self):
+        self.send_tasks = load_script(
+            os.path.join(CWD, '../../scripts'), 'send-tasks'
```
Comment (bjoernricks) [inline tests/scripts/test_send_tasks.py:34]: pathlib is the better choice for handling this.

Comment (y0urself) [reply tests/scripts/test_send_tasks.py:34]: Tell that to python3.5 ;)

File: CHANGELOG.md [hunk 3]
```diff
@@ -20,6 +20,8 @@ $ cd gvm-tools && git log
 * Added tests for `helper` module [#310](https://github.com/greenbone/gvm-tools/pull/310)
 * Added tests for `parser` module [#311](https://github.com/greenbone/gvm-tools/pull/311)
 * Added test for `script/send-target.gmp.py` [#314](https://github.com/greenbone/gvm-tools/pull/314/)
+* Added tests for `send-tasks.py` script [#317](https://github.com/greenbone/gvm-tools/pull/317)
```
Comment (bjoernricks) [inline CHANGELOG.md:23]:
```suggestion
* Added tests for `send-tasks.gmp.py` script [#317](https://github.com/greenbone/gvm-tools/pull/317)
```

File: gvmtools/helper.py [hunk 4]
```diff
@@ -140,6 +187,6 @@ def run_script(path, global_vars):
         file = open(path, 'r', newline='').read()
     except FileNotFoundError:
         print('Script {path} does not exist'.format(path=path), file=sys.stderr)
-        sys.exit(2)
+        quit(2)
```
Comment (bjoernricks) [inline gvmtools/helper.py:190]: Did you miss that line?

**Review by bjoernricks — APPROVED** [review]

**Pull request merged**